data/json/Rome.json

```diff
@@ -1,10 +1,7 @@
 {
   "$schema": "https://raw.githubusercontent.com/dasch-swiss/dsp-meta-svc/main/docs/services/metadata/schema-metadata.json",
```

Comment:
How about I remove the schema instead, it is optional anyway and the api does not return it anymore.

The files are validated against the respective schema depending on project.status in our tests.

I feel it might be an unnecessary burden to the editors if they have to think about choosing the correct schema for no reason.

Reply:
I tend towards the other direction: The schema doesn't hurt for existing files; and while editing this relieves you from having to manually tell the IDE which schema to use (at least VSCode understands the `$schema` property). And from Daniela I hear that she rather copies and adapts existing files than writing them anew (which, admittedly shouldn't happen too often anyways, because for new projects she will use the old GUI for now anyways), and if someone copies an existing file, and the schema is already there, then they don't have to look up where to find the schema in order to get the schema support from their IDE.  
But I do not feel strongly about this, if you prefer to remove it, feel free.